src/dialogs/more-info/controls/more-info-climate.ts

```diff
@@ -197,10 +201,7 @@ class MoreInfoClimate extends LitElement {
         ${supportPresetMode && stateObj.attributes.preset_modes
           ? html`
               <ha-control-select-menu
-                .label=${this.hass.formatEntityAttributeName(
-                  stateObj,
-                  "preset_mode"
-                )}
+                .label=${this.hass.localize("ui.card.climate.preset")}
```

Comment:
Why was this changed?

Reply:
See my response [here](https://github.com/home-assistant/frontend/pull/22043#issuecomment-2453580647).
It's related to remove the word "mode" from the discussion with Matthias above. So in this case the label becomes `Preset` instead of `Preset mode`.

Reply:
Then that should be updated in core, not in frontend? It also seems like a change that can be done in a separate PR?

Reply:
I think that was a bit the point to disconnect the backend state attributes with the frontend translations but ok, let's revert that back to current and let's do/discuss it in a separate PR.

Renaming those state attributes in the backend would be a major overhaul which I don't think we would want.

Reply:
All the related translation changes has been removed from this PR.